### ISSUE TITLE
Label Pop Ups Displaying No Information

### ISSUE THREAD

**Issue by BERNunErnie** [jamesdales/IconMapSupport#74]
I don't know if this feature has been removed as a part of the free version, but when loading in a map with a geojson file from ARCGIS, I can hover over geometries but the Size Element, or Name are not popping up like they were before. They are now just blanks.

**Comment by manouchehr777** [comment]
Yes, I think so. Many features are disabled, without prior notice!
Custom URI, WMS Layer,...

**Comment by chineduocha** [comment]
I'm experiencing the same issue. Does anyone have a fix to this on the free version?

**Comment by Himaanshu-Aggarwal** [comment]
Same here, no tooltips is reflected on when hovering over the size element

**Comment by Marthijs** [comment]
Yes, same issue here. Hoovering over just shows blanks. It worked a couple of weeks ago.

**Comment by jeanny-z8** [comment]
This is a duplicate of this [open ticket](https://github.com/jamesdales/IconMapSupport/issues/71) however it doesn't seem to be going anywhere. The workaround is to create report page tooltips: https://learn.microsoft.com/en-us/power-bi/create-reports/desktop-tooltips?tabs=powerbi-desktop
